src/content/docs/tutorial-app-performance/reduce-javascript-errors.mdx

```diff
@@ -13,56 +13,53 @@ import browserJsErrorCharts from 'images/browser_screenshot-crop_error-overview.
 
-## Finding JavaScript errors [#find-errors] 
+## Objectives [#objectives]
+
+This tutorial walks you through finding and resolve JavaScript errors. By the end of the tutorial, you will have these tasks completed:
+
+* Finding JavaScript errors information in New Relic
+* Applying our recommendations when prioritizing errors
+* Using event logs to resolve JavaScript errors
+
+## Find JavaScript errors [#find-errors] 
+
+From the browser monitoring **Summary** page, two charts can indicate you have JavaScript errors: **Error Rate** and **Top 10 error messages**. Selecting either chart takes you to the **JS errors** page, which gives you a more detailed view of your site'sJavaScript errors. 
+
+* If a customer or internal user reports something that could be a JavaScript error, we recommend starting your search at the **Summary** or **JS errors** pages. 
+* If you want to proactively monitor your site for errors, checking the **Summary** page is adequate most of the time. 
+
+## Determine what errors matter [#determine-errors]
+
+Once you've discovered some JavaScript errors, you need to prioritize your fixes. Prioritizing errors is complex, and can be based on frequency of the error, severity, regions affected, and other factors. Let's use an example to walk you through how to prioritize JavaScript errors. 
 
 <img
     title="JS error charts"
     alt="A screenshot of JavaScript error charts in the browser summary page"
     src={browserJsErrorCharts}
 />
 
-In the <InlinePopover type="browser" /> interface, select your installed monitor to open it's **Summary** page. You'll see two charts, one for **Error Rate** and one for **Top 10 error messages**. Both of these charts can indicate that you have JavaScript errors, and selecting either chart will take you to the **JS errors** page for a more detailed view of JavaScript error specific information. If a customer or internal user reports something that could be a JavaScript error, either the **Summary** page or the **JS errors** page are good places to begin your search. If you just want to monitor your site for errors, checking the **Summary** page will be adequate most of the time. 
-
-## Determining what errors matter [#determine-errors]
-
-Once you've found the JavaScript errors present on your site, you need to determine which ones you need to fix first. The severity of these errors is complex, and it's based on a number of factors, such as frequency, severity, regions affected, and other various factors. 
+Using the **Top 10 error messages** chart above, you can see one error has 6,460 instances. The frequency of this error could make it a prime candidate to resolve first, but that may not always be the case. You may need a wider view of all your errors to determine the order you need to fix them in. From the **JS errors** page, you can group errors by `errorMessage`, `pageURL`, or `deviceType`. Your team should ask themselves these questions when planning their work:
```

Comment:
Austin and I had a discussion about "your team" stuff in these errors. The long and short is that spinning it that way can make it seem like a team is required when we want tutorials to all be something a single user can do. I'd check with him, but I would say you should spin this to "You should keep the following in mind" or something like that.